config/config.default.yaml

```diff
@@ -556,13 +556,16 @@ existing_capacities:
 # docs in https://pypsa-eur.readthedocs.io/en/latest/configuration.html#sector
 sector:
   transport: true
+  transport_emissions: true
```

Comment:
Are these extra emissions switches still necessary? I assume you added them before the CO2 price was integrated to increase the CO2 budget. I suggest removing them in this PR, as they're not related to adding the methane demand.

While having a switch to add emissions from unmodeled sectors (e.g., for carbon budget benchmarking) could be useful, this should be done in a separate PR where we can properly add the emissions as a load to the CO2 atmosphere.

```suggestion
```